content/en/getting_started/tagging/unified_service_tagging.md

```diff
@@ -187,16 +298,62 @@ containers:
               fieldPath: metadata.labels['tags.datadoghq.com/version']
 ```
 
+{{< /collapse-content >}} 
 
 [1]: /agent/cluster_agent/admission_controller/
 [2]: https://kubernetes.io/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information/#capabilities-of-the-downward-api
 [3]: /agent/kubernetes/data_collected/#kube-state-metrics
 [4]: https://github.com/DataDog/integrations-core/blob/master/kubernetes_state/datadog_checks/kubernetes_state/data/conf.yaml.example
 [5]: /tracing/send_traces/
 [6]: /integrations/statsd/
+
 {{% /tab %}}
 
 {{% tab "Docker" %}}
+
+{{< collapse-content title="Set up with automatic version tagging" level="h4" >}}
```

Comment:
Is there a way to make this opened by default when clicking on the Docker tab? It is strange to have to make a choice between automatic and manual version tagging without recommendation

Reply:
I couldn't make it open by default, but I just took it out of the collapsable section to make sure the user can see it. The manual instructions are now collapsed